### DISCUSSION TITLE
[2個問題] 能否從Web Of Science直接抓取多篇論文到Zotero7, Zotero7頁面呈現方式。

### DISCUSSION THREAD

**Discussion by alicebear1203** [zotero-chinese/wiki#207] [求助 / Q&A] [answered]
1. 能否從Web Of Science直接抓取多篇論文到Zotero7？還是只能一篇一篇下載？
2. 如圖所示，我的每一篇論文都會有兩列，一列是正常有項目類型的，一列是附件檔，請問這樣是正常的嗎？
<img width="1110" alt="截圖 2024-06-19 下午5 29 45" src="https://github.com/zotero-chinese/wiki/assets/69949230/d6747ee8-1719-44df-acd0-bc2ee527e27b">

謝謝前輩們協助！ 有你們真好！

**Comment by northword** [comment, marked as answer]
## 问题1

不推荐批量导入，如果一定要批量导入，请确认不会对你的学校造成影响（很多学校会限制批量下载）

导入题录信息即认为导入成功， PDF 附件可能需要逐个下载，或通过检索 PDF ，或通过其他插件。

1. 通过 Connector 导入：

https://zotero-chinese.com/user-guide/add-items#%E9%80%9A%E8%BF%87%E6%B5%8F%E8%A7%88%E5%99%A8%E6%B7%BB%E5%8A%A0%E6%9D%A1%E7%9B%AE

![image](https://github.com/zotero-chinese/wiki/assets/44738481/2be566ae-fb56-4e15-9389-04066b60b70d)




2. 通过导出引文文件载导入 Zotero

![image](https://github.com/zotero-chinese/wiki/assets/44738481/912d79e9-f7ef-4a68-83a8-2e7233bb6a33)

https://zotero-chinese.com/user-guide/add-items#%E7%94%B1%E9%80%9A%E7%94%A8%E6%A0%BC%E5%BC%8F%E7%9A%84%E5%BC%95%E7%94%A8%E4%BF%A1%E6%81%AF%E5%AF%BC%E5%85%A5-%E9%80%9A%E7%94%A8%E6%96%B9%E6%B3%95

## 问题2

不清楚你的情况，如果附件是常规条目的子条目，则是正常的。

**Comment by alicebear1203** [comment]
原來會對學校造成影響！ 我大概需要30篇，但也許我應該循序漸進，先看看摘要，再慢慢導入，不能太貪心。
我會導出paper bibtex，但不知道怎麼導入zotero，我有裝sci-hub！
是個很好的方法，謝謝您！

**Reply by northword** [reply]
30篇应该还好。
有些研究需要大量文献用来做分析，他们可能会一次性导入数百篇上千篇，这种就可能会有问题了。

通过导出通用信息（比如bibtex信息）再导入zotero的这种，理论上是不会有影响的。

**Reply by alicebear1203** [reply]
@northword  好的，謝謝您，那這樣我就安心了。
